sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4130,6 +4132,37 @@ def post_teardown_cleanup(self,
         config = common_utils.read_yaml(handle.cluster_yaml)
         backend_utils.SSHConfigHelper.remove_cluster(handle.cluster_name)
 
+        # Confirm that instances have actually transitioned state before
+        # updating the state database.
```

Comment:
Cna we include the reasoning for not placing this in the instance.py in the comment here?

Reply:
Added more info to the comment.